lab4/src/main/java/lab4/main/Application.java

```diff
@@ -0,0 +1,13 @@
+package lab4.main;
+
+public class Application {
+    public static void main(String[] args) {
+        if (args.length < 2) {
+            System.out.println("Two arguments needed, e.g. file to search and pattern to match");
```

Comment:
"search" should also be included as an argument (args[0]) and there needs to be a check whether arg[0] is search and then continue with the Parser in that case.

Reply:
Has been changed to 3 arguments

Reply:
Looks good!